calculation/seistech_calc/seistech_calc/utils.py

```diff
@@ -418,3 +422,96 @@ def to_mu_sigma(df: pd.DataFrame, im: IM):
     return df.loc[:, [str(im), f"{im}_sigma"]].rename(
         columns={str(im): "mu", f"{im}_sigma": "sigma"}
     )
+
+
+def calculate_gms_spectra(
```

Comment:
This function is only used by the `plot_gms_spectra` function, therefore it should move into the plt_plotting module and become protected (i.e. start with _)
Also something like `prepare_gms_spectra` might be more accurate (since its not actually calculating anything)